alerts/apps_alerts.libsonnet

```diff
@@ -19,7 +19,7 @@
           },
           {
             expr: |||
-              sum by (namespace, pod) (kube_pod_status_phase{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, phase=~"Failed|Pending|Unknown"}) > 0
+              sum by (namespace, pod) (kube_pod_status_phase{%(prefixedNamespaceSelector)s%(kubeStateMetricsSelector)s, phase=~"Failed|Pending|Unknown"} + on(namespace, pod) group_left(owner_kind) 0 * kube_pod_owner{owner_kind!="Job"}) > 0
```

Comment:
why multiply and then add? kube_pod_owner is always 1, so just multiply immediately I'd say

Reply:
I used the trick from https://stackoverflow.com/a/50357418,

I'll test the multiplication

Reply:
Yup works as expected, changed the PR :)